README.md

```diff
@@ -1,129 +1,115 @@
 # Buf for Visual Studio Code
 
-The [VS Code Buf extension](https://marketplace.visualstudio.com/items?itemName=bufbuild.vscode-buf) provides rich support for [Protocol Buffers](https://protobuf.dev/) using the [Buf](https://buf.build/) development environment.
+The [VS Code Buf extension][vs-code-marketplace] provides rich support for [Protocol Buffers][protobuf] using the [Buf][buf] development environment.
 
-## Requirements
+## Features
+
+- **Code navigation** - Go-to definition and documentation insets for `.proto` symbols.
+- **Syntax highlighting** - Protobuf specific color and styling of code.
+- **Code editing** - Formatting via `buf format` and annotations based on `buf lint` and `buf breaking`.
+- **Buf command support** - Execution of `buf` CLI commands via the [Command Palette][command-palette].
 
-- Visual Studio Code
-- A [Buf](https://buf.build/) configuration environment.
+![Preview features](./preview.gif)
```

Comment:
Left a general note above, but just to add here - this Gif is really hard to see. I'm happy to redo it for you if I can get up an running over here.

Reply:
Yes please, would happily take this offer once we get the development environment stuff sorted out!